sdk/ml/azure-ai-ml/setup.py

```diff
@@ -85,6 +85,8 @@
         "azure-common<2.0.0,>=1.1",
         "typing-extensions<5.0.0",
         "opencensus-ext-azure<2.0.0",
+        "azure-mgmt-resourcegraph<9.0.0,>=2.0.0",
```

Comment:
Why added this?

Reply:
This dependency has been removed in last release: #29528 , but we can see we do have dependency to this [p](package: https://github.com/Azure/azure-sdk-for-python/blob/58eee258b24ce90fe1d89afb88d8d93f700d8d06/sdk/ml/azure-ai-ml/azure/ai/ml/_utils/azure_resource_utils.py#L7)

This PR imports sth from `azure_resource_utils.py`, and one dependency checker in CI will raise module not found error. So add this back in this PR, before we know why we removed these before.